src/Microsoft.AspNetCore.Http.Extensions/UriHelper.cs

```diff
@@ -195,17 +195,18 @@ public static string GetEncodedPathAndQuery(this HttpRequest request)
         /// <returns></returns>
         public static string GetDisplayUrl(this HttpRequest request)
         {
-            var host = request.Host.Value;
-            var pathBase = request.PathBase.Value;
-            var path = request.Path.Value;
-            var queryString = request.QueryString.Value;
+            var scheme = request.Scheme.Value ?? "";            
```

Comment:
nit: `string.Empty` vs `""`

I prefer string.Empty but I'm not sure what the standard is for this repo.

Reply:
makes sense, they will both be interned but indeed, it looks like `string.Empty` is used everywhere here